pkg/constants/constants.go

```diff
@@ -130,11 +130,14 @@ var (
 
 // kserve networking constants
 const (
-	NetworkVisibility      = "networking.kserve.io/visibility"
-	ClusterLocalVisibility = "cluster-local"
-	ClusterLocalDomain     = "svc.cluster.local"
-	IsvcNameHeader         = "KServe-Isvc-Name"
-	IsvcNamespaceHeader    = "KServe-Isvc-Namespace"
+	NetworkVisibility       = "networking.kserve.io/visibility"
+	ClusterLocalVisibility  = "cluster-local"
+	ClusterLocalDomain      = "svc.cluster.local"
+	IsvcNameHeader          = "KServe-Isvc-Name"
+	IsvcNamespaceHeader     = "KServe-Isvc-Namespace"
+	ODHKserveRawAuth        = "security.opendatahub.io/enable-auth"
```

Comment:
This constant is already defined, please reuse it:

```
	LabelEnableAuthODH = "security.opendatahub.io/enable-auth"
```

Reply:
I don't see that in this repo. `LabelEnableAuthODH` is in odh-model-controller I think

Reply:
oops, yes, haha